model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -364,6 +348,7 @@ def copy_graph_with_ops(graph: Graph) -> Graph:
                 log.warning('Operation {} is not found in MO operations, please check it! '
                             'Simple shape infer function is used'.format(op_type))
                 node = Op(new_graph, op.attrs()).create_node()
+                node['op'] = node.soft_get('type')
```

Comment:
In this case `node.soft_get('type')` is equal to `node.type`. I prefer the the second variant, because we do not have a default value for the attribute `type`. But if something bad happens and `type` is `None`, `soft_get` will return `<UNKNOWN>`. What do you think, will it be a problem during debugging?
I am against using `soft_get` in every case, when we want to get an attribute value.

Reply:
Yes, it will. It is better to use assert and check value to avoid `<UNKNOWN>` value of `node.op`. Even if `node.type` always exist in this case. I will update this.

Reply:
Done.